box/config.yml

```diff
@@ -1271,7 +1275,7 @@ php_packages_extra:
 # XDebug configuration.
 php_xdebug_default_enable: 1
 php_xdebug_cli_disable: no
-php_xdebug_remote_autostart: 1
+php_xdebug_remote_autostart: 0
```

Comment:
We don't want to commit this. Should stay 1

Reply:
@joewhitsitt agh I forgot it automatically commits any changed files while doing provisions! Thank you so much!